etl/steps/data/garden/demography/2022-11-30/life_expectancy.py

```diff
@@ -395,3 +435,50 @@ def add_region_aggregates(frame: pd.DataFrame) -> pd.DataFrame:
     # concatenate
     df = pd.concat([frame, df]).sort_values(["country", "year"], ignore_index=True).drop(columns="population")
     return df
+
+
+def add_population_americas(df: pd.DataFrame):
+    pop = load_america_population_from_unwpp()
+    df = df.merge(pop, on=["country", "year"])
+    return df
+
+
+def load_america_population_from_unwpp():
+    """Load population data from UN WPP for Northern America and Latin America and the Caribbean.
+
+    We use this dataset instead of the long-run because we want the entities as defined by the UN.
+    """
+    # load population from WPP
+    locations = ["Latin America and the Caribbean (UN)", "Northern America (UN)"]
+    ds = Dataset(GARDEN_POPULATION_WPP)
+    df = ds["population"].reset_index()
+    df = df.loc[
+        (df["location"].isin(locations))
+        & (df["metric"] == "population")
+        & (df["sex"] == "all")
+        & (df["age"] == "all")
+        & (df["variant"].isin(["estimates", "medium"])),
+        ["location", "year", "value"],
+    ]
+    assert len(set(df["location"])) == 2, f"Check that all of {locations} are in df"
+    df["location"] = df["location"].replace(REGION_MAPPING)
+
+    # rename columns
+    df = df.rename(columns={"location": "country", "value": "population"})
+    return df
+
+
+def make_anomaly_table() -> Table:
+    log.info("life_expectancy: making anomaly table")
+    # Load anomaly yaml file
+    with open(PATH_ANOMALIES) as f:
+        anomalies = yaml.safe_load(f)
+    # store all yaml's content as a string in a cell in the table
+    df = pd.DataFrame({"anomalies": [str(anomalies)]})
+    tb = Table(df)
+    # add metadata
+    tb.metadata = TableMeta(
+        short_name="_anomalies",
+        description="this table contains the anomalies for the life expectancy data in YAML format.",
```

Comment:
I just want to point out that what you are providing here are not anomalies. They are possible causes of the anomalies. The anomalies themselves would be simply data points (e.g. France 1918). Proving that those strange points are caused by those events would require some complicated causal inference. So, somewhere, you could make that clarification (maybe in the grapher step, where you list the events).

Reply:
changed naming of files to "historical events"